scripts/Entity.gd

```diff
@@ -26,6 +28,9 @@ func jump():
 	motion.y = -jump_force
 	resistance = AIR_RESISTANCE
 
+func take_damage():
```

Comment:
There should be a common code for taking damage, like `lives -= 1`.

Also, this method should receive an `int` variable as a parameter saying the damage value.